tests/ui/test_other_user.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.ui.mcg_ui import ObcUI
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.helpers.helpers import create_unique_resource_name
+from ocs_ci.ocs.ui.views import locators
+from ocs_ci.framework.testlib import ManageTest, ui, bugzilla, polarion_id, tier2
+from time import sleep
+from ocs_ci.utility.utils import ceph_health_check
+from tests.conftest import delete_projects
+
+logger = logging.getLogger(__name__)
+
+
+class ObcUi(ObcUI):
```

Comment:
Can you move this class to mcg_ui.py ?
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/ui/mcg_ui.py